styles/duckduckgo/catppuccin.user.css

```diff
@@ -659,12 +657,28 @@
       background-color: @blue !important;
       color: @crust !important;
     }
+    .modal--dropdown--settings
+      .settings-dropdown--section
+      .settings-dropdown--header
+      .settings-dropdown--header--label,
+    .modal--dropdown--settings
+      .settings-dropdown--section
+      .settings-dropdown--header
+      .settings-dropdown--header--link {
+      color: @text !important;
```

Comment:
you could use some nesting here to save you typing out the same lines a few times.